app/controllers/users_controller.rb

```diff
@@ -5,6 +5,5 @@ def index
 
```

Comment:
- [ ] It would be better if we add the likes controller actions so we can fully implement the functionality. Remember to add a successful and failure message depending on the results of the like variable

Reply:
Fixed